### ISSUE TITLE
[CAL-1079] Should seated events be grouped together at display?

### ISSUE THREAD

**Issue by zomars** [calcom/cal.com#7127]
Currently each "seat" is displayed as an individual booking which is not incorrect but it feels weird. shouldn't we do something to distinguish this specific slot? cc @Jaibles 

![image.png](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/c4ec01bb-487d-4809-99f6-4dbb2109aa0d/a103bf7c-41f4-4c70-8698-445c49baee24)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1079](https://linear.app/calcom/issue/CAL-1079/should-seated-events-be-grouped-together-at-display)</sub>

**Comment by PeerRich** [comment]
wait why should someone have multiple seats?

**Comment by Jaibles** [comment]
IMO as an organiser, 1 booking with 10 seats is still just 1 booking.

**Seated event**
5/10 seats booked

**Comment by PeerRich** [comment]
ohh this is on the receiving end. yes we need to group them. maybe as an accordion?

**Comment by ologbonowiwi** [comment]
We lost track of this one, but it looks like there's a current behavior that relates to this issue.
<img width="1605" alt="image" src="https://github.com/calcom/cal.com/assets/100464352/872fb769-4569-4cff-a6f0-ee8952859ded">

Steps to reproduce this card/behavior:
- Create an event with multiple seats
- Get two bookings on the same slot with different e-mails (here you'll see different bookings on `/bookings/upcoming`)
- Reschedule the first booking to another slot
- Reschedule the second booking to the same slot as the first one

After following these steps, you'll see only one booking on the `/bookings/upcoming` list, but it'll show the name of all attendees.

From what I found working on #10631, this behavior is achieved by the method `handleSeats` on [`/packages/features/bookings/lib/handleNewBooking.ts:1111`](https://github.com/calcom/cal.com/blob/main/packages/features/bookings/lib/handleNewBooking.ts#L1111)

Although, one thing that I noticed is that the title of the booking it'll be "{{hostName}} and {{firstAttendee}}".
Another concern is that we must account for hiding other attendees in the current implementation. If you follow the steps I mentioned, everyone besides the first attendee sees the other attendees in the same slot.
